cilium/network_policy.cc

```diff
@@ -1104,17 +1104,16 @@ NetworkPolicyMap::NetworkPolicyMap(Server::Configuration::FactoryContext& contex
     : context_(context.serverFactoryContext()), tls_map_(context_.threadLocal()),
       local_ip_str_(context_.localInfo().address()->ip()->addressAsString()),
       name_(fmt::format("cilium.policymap.{}.{}.", local_ip_str_, ++instance_id_)),
```

Comment:
can we  clean this up even further and remove the instance variables `name_` & `instance_id_`. 

Looks like now the name is only used in a log statement where it could be inlined (dynamic part (IP & ID) shouldn't be of relevance there too)

Reply:
instance id has been informative before, so I'll keep it

Reply:
as a result of removing `name_` we could also get rid of  the field `local_ip_str_` :thinking:

Reply:
Updated